plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/edit.tsx

```diff
@@ -137,32 +136,21 @@ const Edit = ( props: EditProps ) => {
 			allowedBlocks: getAllowedBlocks( EXCLUDED_BLOCKS ),
 			template: [
 				[
-					'core/group',
+					'core/heading',
 					{
-						layout: {
-							type: 'flex',
-							flexWrap: 'nowrap',
-						},
-						metadata: {
-							name: __( 'Header', 'woocommerce' ),
-						},
+						level: 4,
+						content:
+							attributeObject?.label ||
+							__( 'Attribute', 'woocommerce' ),
 						style: {
 							spacing: {
-								blockGap: '0',
+								margin: {
+									bottom: '10px',
+									top: '0',
+								},
```

Comment:
Nitpick: I think it's better to use `rem` units instead of `px` for the margin values. Using `rem` units would better support Accessibility and responsive design.